tcms/management/management/commands/baseurl.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+
+
+class Command(BaseCommand):
+    help = "Sets the base URL of KiwiTCMS instance. " \
+           "If no arguments given returns current base URL."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'baseurl', nargs='?', default=None,
+            help='Base URL of KiwiTCMS instance',
+        )
+        parser.add_argument(
+            'basename', nargs='?', default='KiwiTCMS',
+            help='Optional site name. Defaults to "KiwiTCMS"',
+        )
+
+    def handle(self, *args, **kwargs):
+        if not kwargs['baseurl']:
```

Comment:
Shouldn't nargs + eliminate the need for this ?

Reply:
Yes, but this will make the argument mandatory. My idea is to print current Base URL if no argument is specified.